app/src/main/java/org/oppia/app/recyclerview/DivideItemDecorator.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.app.recyclerview
+
+import android.graphics.Canvas
+import android.graphics.drawable.Drawable
+import androidx.recyclerview.widget.RecyclerView
+
+class DividerItemDecorator(private val mDivider: Drawable) : RecyclerView.ItemDecoration() {
```

Comment:
Hmm, could we add the separate drawable as another view type in the recycler view itself? Given that we aren't dragging and dropping elements within groups, that's more of what I would expect. While it doesn't matter for this particular recycler view since the number of elements will be small, this is technically potentially drawing the divider off screen for children that aren't fully visible on the screen.

Reply:
we can add another view type but that would defeat the purpose of item decorator also we would need to add/remove items manually which can cause issues especially being a triple nested recyclerview

Reply:
How does the triple recycler view complicate this? I do expect that we need to manage the list manually. Does ``childCount`` below correspond to the number of visible children or all children in the recycler view? I'm assuming the former, in which case I think the decorator is fine, I just want to be sure I understand this implementation.

Reply:
Per offline discussion, deciding to keep the recycler view simpler for now. We can always change this later if there's a need.